src/iframeMessenger.js

```diff
@@ -423,6 +423,47 @@
             }
         }
 
+        function _addAcquisitionDataToLinks(acquisitionData) {
+            var links = document.getElementsByClassName('js-acquisition-link');
+            var json = encodeURIComponent(JSON.stringify(acquisitionData));
+            for (var i = 0; i < links.length; i++) {
+                var href = link.attr('href');
```

Comment:
I think this should be `.setAttribute()`

https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute